.jenkins/pipelines/scheduled-tests.Jenkinsfile

```diff
@@ -0,0 +1,173 @@
+/* A Jenkins pipeline that will handle code coverage and nightly tests
+*  These are the original pipelines:
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-code-coverage-nightly.yml
+*/
+
+pipeline {
+    agent {
+        label 'ACC-1804-DC4'
+    }
+    options {
+        timeout(time: 600, unit: 'MINUTES')
```

Comment:
we might only need a 300 min timeout - afraid if a 10 hr timeout would cause extra resource usage in case something is stuck?